src/Elcodi/Bundle/AttributeBundle/CompilerPass/MappingCompilerPass.php

```diff
@@ -26,11 +26,7 @@
 class MappingCompilerPass extends AbstractMappingCompilerPass
 {
     /**
-     * You can modify the container here before it is dumped to PHP code.
```

Comment:
I'll readd it later today.

In my opinion that description wasn't helping much for developers who are
learning Elcodi as it refers to default Symfony Dependency Injection
functionality.

On Wednesday, March 18, 2015, Marc Morera <notifications@github.com> wrote:

> In src/Elcodi/Bundle/AttributeBundle/CompilerPass/MappingCompilerPass.php
> <https://github.com/elcodi/elcodi/pull/612#discussion_r26671039>:
>
> > @@ -26,11 +26,7 @@
> >  class MappingCompilerPass extends AbstractMappingCompilerPass
> >  {
> >      /**
> > -     * You can modify the container here before it is dumped to PHP code.
>
> Even if it is copied from the other method, this text is the textual
> representation of what this method is intended for, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elcodi/elcodi/pull/612/files#r26671039>.
>